src/md/ceefilegen/cceegen.cpp

```diff
@@ -610,7 +610,7 @@ HRESULT CCeeGen::emitMetaData(IMetaDataEmit *emitter, CeeSection* section, DWORD
     
     if (! m_fTokenMapSupported) {
         // Remove the handler that we set
-        hr = emitter->SetHandler(NULL);
+        emitter->SetHandler(NULL);
```

Comment:
This cannot actually fail. Other place in the file that calls this with NULL ignores the HRESULT too.